CONTRIBUTING.md

```diff
@@ -85,6 +85,17 @@ Note that we use PEP 484 type hints, so parameter types should be removed from t
 }
 ```
 
+### PySpark tests
+PySpark tests are invoked separately from the rest since they require
+installing Java and the large PySpark package.
+They are executed on Travis, but not by default for a local `tox` command.
```

Comment:
repeating discussion we had earlier, TBD a way to not install packages every time a PR is created/updates are pushed to Travis

Reply:
You mean the `tox` setup steps on Travis are taking too long?

Reply:
yeah...seems like caching pip (which we already do) is the best we can do. i can keep digging.

Reply:
Opened a task in Asana, pushing on this now.